src/pages/Folder/FolderPage.styled.js

```diff
@@ -0,0 +1,118 @@
+import styled from 'styled-components';
+
+export const FolderPageLayout = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export const FolderPageHeader = styled.header`
+  background: var(--color-gray-bg);
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  padding: 60px 23px 90px;
+`;
+
+export const HeaderSearch = styled.div`
+  width: 800px;
+`;
+
+export const FolderPageContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 1060px;
+  padding: 40px 32px 100px 32px;
+
+  /* Tablet */
+  @media (max-width: 1124px) {
+    & {
+      width: 100%;
+    }
+  }
+`;
```

Comment:
FolderPage.styled.js로 styled로 만든 컴포넌트를 분리해서 FolderPage.jsx에서 조립하고 있습니다.
styled 컴포넌트를 만들 때 접두사로 컴포넌트 이름을 붙여서 FolderPage~로 만들어봤는데 갈수록 이름이 길어지고 보기가 안 좋더라구요.
혹시 재사용이 아닌 FolderPage만을 위한 컴포넌트라면 접두사를 빼고 구성해도 괜찮을까요?

```suggestion
export const Layout = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
`;

export const Header = styled.header`
  background: var(--color-gray-bg);
  display: flex;
  justify-content: center;
  width: 100%;
  padding: 60px 23px 90px;
`;

export const HeaderSearch = styled.div`
  width: 800px;
`;

export const Content = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 1060px;
  padding: 40px 32px 100px 32px;
  /* Tablet */
  @media (max-width: 1124px) {
    & {
      width: 100%;
    }
  }
`;
```

Reply:
그것도 스타일입니다! 다만, 길다고 가독성이 좋지 않는 건 없습니다. (정말 무의미한 게 아니라면)